libs/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "compilerOptions": {
-    "module": "CommonJS",
-    "target": "ESNext",
+    "moduleResolution": "bundler",
+    "module": "es2020",
```

Comment:
Is the `es2020` choice deliberate? If so, why not a more recent version?

Reply:
I used the oldest version that supported the import.meta features we needed, but we could also use a more recent version. LMK what you think we should do on that.